### PR TITLE
LPS-65857 Add Auth and Encryption to JGroups configuration

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test:clustering

**Comment by dantewang** [comment]
@zhangyan0701 Please prioritize this PR, thanks!

**Comment by zhangyan0701** [comment]
No regressions found.

File: portal-impl/src/portal.properties [hunk 1]
```diff
@@ -7229,29 +7236,29 @@
     feature.flag.LPS-194362.type=release
 
     #
-    # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASES__MINUS__NUMBER1__NUMBER9__NUMBER6__NUMBER7__NUMBER2__NUMBER4__PERIOD_DESCRIPTION
+    # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASES__MINUS__NUMBER1__NUMBER9__NUMBER4__NUMBER7__NUMBER6__NUMBER3__PERIOD_DESCRIPTION
```
[the remark this answers is not in the document]
Comment (julschong) [reply portal-impl/src/portal.properties:7239]: split into this commit https://github.com/liferay-core-infra/liferay-portal/pull/6281/commits/6759908819a0901d4fd5d952f2550ba13de844b7

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
@tinatian I will add warning message and add non-empty default passcode and resubmit

**Pull request closed without merging**